python-lib/amazon_transcribe_api_client.py

```diff
@@ -318,3 +321,25 @@ def _result_parser(self,
             logging.warning(f"Unknown state encountered: {job_status}")
             raise UnknownStatusError(f"Unknown state encountered: {job_status}")
         return job_data
+
+    @staticmethod
+    def handle_job_duration(job_summary: Dict,
+                            job_res_data: Dict):
+        """
+        If the job duration is longer than the timeout setup previously, a JOB_TIMEOUT_ERROR will
+        be written in the column error of the Dataframe, otherwise it returns None.
+        """
+
+        date_job_created = job_summary.get("CreationTime")
+        now = datetime.datetime.now(tz=date_job_created.tzinfo)
+        time_delta_sec = (now - date_job_created).seconds
+        time_delta_min = time_delta_sec / 60
```

Comment:
no need to store seconds here, directly assign to minutes

Reply:
done in 88b96bbe1935d7840e46260a3c3ddc5be2ad7b0c